packages/analytics/package.json

```diff
@@ -1,32 +1,31 @@
 {
     "scripts": {
         "lint": "run -T eslint . --ext .ts,.tsx",
-        "clean": "rm -rf ./dist ./esm",
+        "clean": "rm -rf ./dist",
         "dev": "tsc --watch",
-        "build": "tsc && npm run build:esm",
-        "build:esm": "tsc -p ./tsconfig-esm.json",
+        "build": "run -T tsup src/index.ts --format cjs,esm --dts && run -T publint",
```

Comment:
`run -T` is [yarn syntax](https://yarnpkg.com/getting-started/qa#how-to-share-scripts-between-workspaces) that allows you to run packages that are installed in the root of the monorep